src/rpc/mining.cpp

```diff
@@ -805,10 +808,11 @@ UniValue estimatesmartfee(const JSONRPCRequest& request)
             + HelpExampleCli("estimatesmartfee", "6")
             );
 
-    RPCTypeCheck(request.params, {UniValue::VNUM, UniValue::VSTR});
+    RPCTypeCheck(request.params, {UniValue::VNUM, UniValue::VSTR, UniValue::VBOOL});
```

Comment:
This should make null estimate_mode args more relevant, which would make #12940 or something like it more important to include.